front/src/modules/ui/input/components/EntityTitleDoubleTextInput.tsx

```diff
@@ -25,7 +25,7 @@ const StyledDoubleTextContainer = styled.div`
 const StyledTextInput = styled(StyledInput)`
   margin: 0 ${({ theme }) => theme.spacing(0.5)};
   padding: 0;
-  width: ${({ width }) => (width ? `${width}px` : 'auto')};
+  width: ${({ width }) => (width ? `calc(${width}px + 1.5%)` : 'auto')};
```

Comment:
I don't think this is the right fix, see on front app where I've tested it:
<img width="334" alt="image" src="https://github.com/twentyhq/twenty/assets/12035771/9fabb113-3e8f-4a34-b54f-185799a78e31">

I have investigated and it seems that while the page is loading the shadow element is changing is width from 61 to 64 but the change is not propagated. This is because the useEffect() dependencies from `ComputeNodeDimensions` are not right

Let's replace the ComputeNodeDimensions code by:
```
import { ReactNode, useEffect, useRef, useState } from 'react';
import styled from '@emotion/styled';

type ComputeNodeDimensionsProps = {
  children: (
    dimensions: { height: number; width: number } | undefined,
  ) => ReactNode;
  node?: ReactNode;
};

const StyledNodeWrapper = styled.span`
  pointer-events: none;
  position: fixed;
  visibility: hidden;
`;

export const ComputeNodeDimensions = ({
  children,
  node = children(undefined),
}: ComputeN
```odeDimensionsProps) => {
  const nodeWrapperRef = useRef<HTMLSpanElement>(null);
  const [nodeDimensions, setNodeDimensions] = useState<
    | {
        width: number;
        height: number;
      }
    | undefined
  >(undefined);

  useEffect(() => {
    if (nodeWrapperRef.current) {
      nodeWrapperRef.current &&
        setNodeDimensions({
          width: nodeWrapperRef.current.offsetWidth,
          height: nodeWrapperRef.current.offsetHeight,
        });
    }
  }, [nodeWrapperRef?.current?.offsetWidth]);

  return (
    <>
      <StyledNodeWrapper ref={nodeWrapperRef}>{node}</StyledNodeWrapper>
      {nodeDimensions && children(nodeDimensions)}
    </>
  );
};